spec/aiport_spec.rb

```
@@ -0,0 +1,81 @@
+require 'airport'
+require 'plane'
+describe Airport do
+
+  it 'responds to land method' do
+    allow(subject).to receive(:bad_weather?) { false }
+    expect(subject).to respond_to :land
+  end
+
+  it 'responds to take_off method' do
+    allow(subject).to receive(:bad_weather?) { false }
+    expect(subject).to respond_to :take_off
+  end
+end
+
+describe '#land' do
+  airport = Airport.new
+  plane = Plane.new
+
+  it 'lands a plane' do
+    allow(airport).to receive(:bad_weather?) { false }
+    expect(airport.land(plane)).to eq [plane]
+  end
+
+  it 'requires 1 argument' do
+    expect(airport).to respond_to(:land).with(1).argument
+  end
+
+  it 'raises an error when full' do
+    allow(airport).to receive(:bad_weather?) { false }
+    # this is maximum length -1 as we've already landed a plane in line 19!!
+    9.times { airport.land Plane.new }
```

May be silly question!, but if capacity is 10 why would you want to raise an error after running the loop 9 times?